lib/spack/spack/test/cmd/resource.py

```diff
@@ -21,6 +24,9 @@
     '7d865e959b2466918c9863afca942d0fb89d7c9ac0c99bafc3749504ded97730',
 ]
 
+pytestmark = pytest.mark.skipif(sys.platform == "win32",
+                                reason="does not run on windows")
+
```

Comment:
I think we'll want to be able to test the most basic resource behavior on Windows. Or do resources in fact not work?

Reply:
All tests fail, seemingly due to a lack of, or a conflict with, the hashes in the resource output. I'm unsure of the cause at this time. We can try to get this passing prior to merge if it's something you think it imperative to have on Windows as soon as it's merged.